packages/react-components/react-teaching-popover-preview/src/components/TeachingPopoverSurface/renderTeachingPopoverSurface.tsx

```diff
@@ -1,15 +1,27 @@
-/** @jsxRuntime automatic */
-/** @jsxImportSource @fluentui/react-jsx-runtime */
+/** @jsxRuntime classic */
+/** @jsx createElement */
 
+import { createElement } from '@fluentui/react-jsx-runtime';
+import { Portal } from '@fluentui/react-portal';
+import type { TeachingPopoverSurfaceSlots, TeachingPopoverSurfaceState } from './TeachingPopoverSurface.types';
 import { assertSlots } from '@fluentui/react-utilities';
-import type { TeachingPopoverSurfaceState, TeachingPopoverSurfaceSlots } from './TeachingPopoverSurface.types';
 
 /**
- * Render the final JSX of TeachingPopoverSurface
+ * Render the final JSX of PopoverSurface
  */
 export const renderTeachingPopoverSurface_unstable = (state: TeachingPopoverSurfaceState) => {
   assertSlots<TeachingPopoverSurfaceSlots>(state);
 
-  // TODO Add additional slots in the appropriate place
-  return <state.root />;
+  const surface = (
+    <state.root>
+      {state.withArrow && <div ref={state.arrowRef} className={state.arrowClassName} />}
```

Comment:
Should `ref` and `className` be set in hooks?

Reply:
I think it makes sense for this to be in the hook, right now it's just matching the implementation of Popover

Reply:
Updated: Modified this in the base react-popover class to be an optional arrow slot that gets defined in the hooks instead.